controllers/quay/quayregistry_controller.go

```diff
@@ -747,6 +747,13 @@ func (r *QuayRegistryReconciler) updateWithCondition(
 	}
 	quay.Status.Conditions = v1.SetCondition(quay.Status.Conditions, condition)
 	quay.Status.LastUpdate = time.Now().UTC().String()
+
+	eventType := corev1.EventTypeNormal
+	if cstatus == metav1.ConditionTrue {
+		eventType = corev1.EventTypeWarning
+	}
+	r.EventRecorder.Event(quay, eventType, string(reason), msg)
```

Comment:
Have you added this for debug purposes?

Reply:
I've removed this as we should just report in the status controller.